DTR.md

```diff
@@ -0,0 +1,61 @@
+## Project: Futbol
+
+### Group Member Names: Brian Greeson, David Tran, Tony Gomes, Krista Stadler
+
+### Goals and Expectations for the Project (What does each group member hope to get out of this project? What do we want to achieve as a team? How will we know that we're successful?):
+
+Our goals are to learn how to build something in a team while properly using version control.
+We want to focus on collaboration and working with others -- we will know we are successful when we have working program that was written collaboratively.
+
+### Team strengths & collaboration styles (consider discussing your Pairin qualities here):
+
+Brian is technically proficient and happy answers questions over slack and in person.
+Tony wants to focus on being a good follower during this project and wants to champion the data piece of the project.
+Krista is well organized and will take the lead on project management.
+
+### Schedule Expectations (When are we available to work together and individually? What constraints do we have?):
+Krista's availability is M-F 8:45am - 5pm and 8pm - 10pm
+Saturday 12pm-5pm and 8pm-10pm
+Sunday 9am-5pm and 8pm-10pm
+
+David's schedule is flexible and he can meet any time.
+
+Tony's availability is M-F 8-12 & 1-5 & 7-9
+Sat-Sun 9-12 or 1-5
+
+Brian's availability is M-F 8-12 & 1-5 & 6:30-10
+Sat-Sun 12- 11
+
+
+We'll have a daily standup at 8:45am.
+We'll have group check in and work time from 8pm - 9pm.
+
+
+### Communication Expectations (How and often will we communicate? How do we keep lines of communication open? How will we make decisions as a team?):
+
+Our group expectation is that during times you're committed to working on the project, slack response is expected within 5 minutes -- at the very least, you should send a quick message to let the sender know when you will respond.
+
+Tony prefers text messaging. We will all share our phone numbers but we do all expect good phone etiquette -- no calling or texting after 9pm.
```

Comment:
Slack works for me actually. We can update this on the next retro.

Reply:
I'm really confused on how this file got tracked by git.. i'll remove it from this branch as it's not relevant